### DISCUSSION TITLE
Stop trying to be Facebook

### DISCUSSION THREAD

**Discussion by tuupola** [community/community#13405] [Other Feature Feedback, Questions, & Ideas]
Stop trying to be a social media site. Stop trying to add algorithm which guess what I am interested in. Just give me the raw feed of people and repositories I am subscribed to.

**Comment by NitinIsFit** [comment]
Yeah, but instead of removing the "for you" page, GitHub can just add an option "turn off algorithmic feed", but tbh i want to see better version of for yo page, i wanna add a list of certain keywords and want to block certain keywords on for you page so GitHub could recommend me content accordingly.

 ![Screenshot_20220727-112102.png](https://user-images.githubusercontent.com/94877880/181171208-4ac6b69e-6fd3-452c-a76a-57761670dbc0.png)

![Screenshot_20220727-112117.png](https://user-images.githubusercontent.com/94877880/181171229-103fa77f-3308-46c4-96b6-c08f51fc74f4.png)

This is something I am talking about.

**Comment by jfullstackdev** [comment]
yup, as for me, there should be an option to disable features, 

as of now, there is the option to hide the badges and make the profile private

but it does not mean GitHub will stop adding features, others love them too

**Comment by willpower232** [comment]
I've interacted with a lot of people in various repos and I'm definitely not interested in what they're up to if I haven't already followed them.

**Reply by Matsa59** [reply]
at least it suggests you more than 1 user ... Github only suggests me 1 guy that I don't know ...

**Reply by NitinIsFit** [reply]
> at least it suggests you more than 1 user ... Github only suggests me 1 guy that I don't know ...

...Lmaoo
